app/code/community/Oyst/OneClick/Model/Order/ApiWrapper.php

```diff
@@ -81,4 +82,21 @@ public function updateOrder($oystOrderId, $status)
 
         return $response;
     }
+
+    /**
+     * Refund an order.
+     *
+     * @param $orderId
+     * @param null $price
+     */
+    public function refund($orderId, $price = null)
+    {
+        try {
+            $price = new OystPrice($price, 'EUR');
+            $this->_orderApi->refunds($orderId, $price);
+
```

Comment:
Could you use the workflow of:
```
$response =  $this->_orderApi->refunds($orderId, $price);
$this->_oystClient->validateResult($this->_orderApi);
```

And return the ```$response``` to allow to get the information about the refunds